### DISCUSSION TITLE
Possibility of mapping to control by midi controllers?

### DISCUSSION THREAD

**Discussion by Maneleki** [upstage-org/upstage#1136] [Ideas]
Dear devs and other criatures! 
I wonder about the possibility to map the tools, controls, scenes, or whatever is possible for midi controllers.
In different software like Ableton, Resolume, Adobe Premiere... you have the possibility to use midi controllers in order to control different aspects of it. And it helps a lot to do live perfomances. So I was wondering if it is possible to implement it in UpStage.

**Comment by helenvarley** [comment]
this idea has come up before but never been deeply explored ... it would definitely open up a lot of possibilities, especially if a player could do more than one thing at a time (as with mouse/keyboard/touchscreen)

**Comment by vickis** [comment]
Worth exploring yes :)
